### DISCUSSION TITLE
Инструментарий по работе с профилями\руководствами (igpop, zen lang | sushi, fhir shorthand).

### DISCUSSION THREAD

**Discussion by Vrodvi** [fhir-ru/core#29] [General]
Объявлено об использовании инструментария igpop и zen lang.

Насколько данный инструментарий соотносится по функциональным возможностям с официальным инструментарием, предлагаемым HL7 - [fhir shorthand](http://build.fhir.org/ig/HL7/fhir-shorthand/reference.html) и [sushi](https://fshschool.org/)?

Если функциональные возможности сопоставимы, то предлагаю собрать в этом топике ссылки на документацию по igpop и zen lang, аналогично документации по fhir shorthand.

Как минимум, хотелось бы иметь:
- Документацию по igpop (по аналогии с документацией по [SUSHI](https://fshschool.org/docs/))
- Документацию по zen lang (по аналогии с документацией по [FHIR Shorthand](http://build.fhir.org/ig/HL7/fhir-shorthand/reference.html))

Дополнительно, но желательно:
- Обучающие материалы по работе с igpop (по аналогии с [tutorials](https://fshschool.org/docs/tutorials/)) 
- Репозиторий примеров (по аналогии с [fsh-finder](https://fshschool.org/fsh-finder/))
- Конвертер Zen<->FHIR (по аналогии с [конвертер FSH<->FHIR](https://fshschool.org/FSHOnline/#/))

**Comment by niquola** [comment]
Добавил инструкции по локальной сброке - https://github.com/fhir-ru/core#%D0%B2%D0%BA%D0%BB%D0%B0%D0%B4-%D0%B8-%D1%80%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B0
#27 

@Semalexa  у нас есть документация по igpop - если нет - давай напишем базовую

**Comment by Vrodvi** [comment]
Представленный на совещании вариант руководства по реализации (IG), составленный с помощью инструмента igpop, отличается от стандартного формата IG, составляемого с помощью HL7 publisher'а.
Например, в представленном руководстве нет возможности скачать ресурс-профиль и примеры ресурсов соответствующих профилю.

Пример описания организации из US Core - https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-organization.html,
и из RU Core - https://fhir-ru.github.io/core/profiles/Organization/base.html.
Информации и функциональности в RU Core явно меньше.

Есть ли возможность формировать руководство в стандартном формате с помощью HL7 publisher'а?

---
По данному поводу также имеется не закрытая issue: https://github.com/fhir-ru/core/issues/27
